app/views/layouts/_footer.html.erb

```diff
@@ -1,7 +1,7 @@
 <footer class="footer">
   <small>
     <%= t ".the" %> <%= link_to t(".rails"), "http://www.railstutorial.org/" %>
-    <%= t ".by" %> <%= link_to "Michael Hartl", "http://www.michaelhartl.com/" %>
+    <%= t ".by" %> <%= link_to t(".author"), "http://www.michaelhartl.com/" %>
```

Comment:
No need to use I18n for the author name, can you write your name in Bangla font ? :))